worker/featureflag/package_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package featureflag_test
+
+import (
+	"testing"
+
+	coretesting "github.com/juju/juju/testing"
+)
+
+func TestPackage(t *testing.T) {
+	coretesting.MgoTestPackage(t)
```

Comment:
There's no mgo needed

Reply:
It's needed for StateSuite (which we need because the manifold depends on the state worker).